test/test_wsgi.py

```diff
@@ -12,8 +12,6 @@ def tflow():
 
 class TestApp:
 
-    def __init__(self):
-        self.called = False
```

Comment:
Py.test gave a warning that it could not collect TestApp because there was an __init__ constructor. 

Maybe this warning isn't an issue for the tests. I'm not familiar with pytest. I decided to err on the side of no warnings. 

Thanks,
Sam
Sent from my iPhone

> On Nov 11, 2015, at 11:54 PM, Maximilian Hils <notifications@github.com> wrote:
> 
> In test/test_wsgi.py:
> 
> > @@ -12,8 +12,6 @@ def tflow():
> >  
> >  class TestApp:
> >  
> > -    def __init__(self):
> > -        self.called = False
> Why this change? 
> 
> —
> Reply to this email directly or view it on GitHub.
>